resources/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
               name: log4j2-vol
             - mountPath: /home/wso2/lib/dropins
               name: dropins-vol
-          image: wso2/choreo-connect-enforcer:0.9.0.2
+          image: wso2/choreo-connect-enforcer:0.9.0
```

Comment:
These versions should be snapshot. Check all the places as well

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2000/commits/24871c5ba5b2c1fba5acf2c97ef707f9fd591787